src/services/EdFi.DataManagementService.Api/Core/Handler/DeleteByIdHandler.cs

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.DataManagementService.Core.Pipeline;
+
+namespace EdFi.DataManagementService.Api.Core.Handler;
+
+/// <summary>
+/// Handles a delete request that has made it through the middleware pipeline steps.
+/// </summary>
+public class DeleteByIdHandler : IPipelineStep
```

Comment:
Handlers are the final step in the pipeline. They are where core will interface with the backend DB plugins through a to-be-defined interface.